src/pages/builders/index.ts

```diff
@@ -395,7 +395,7 @@ export class PlaywrightScriptBuilder extends ScriptBuilder {
   };
 
   awaitText = (text: string) => {
-    this.pushCodes(`await page.waitForSelector('text=${text}')`);
+    this.pushCodes(`await page.waitForSelector('text=${text}');`);
```

Comment:
Cypress official style guide doesn't encourage semicolon https://github.com/cypress-io/cypress-documentation/issues/853 and that's why I didn't add it (all their example codes don't have semicolon)

Reply:
It looks like while the Cypress team does not use semicolons, looking at a sampling of Cypress scripts on Github that I quickly looked at, it looks like the vast majority of users still use semicolons in their test scripts.

- https://github.com/psurmicki/stx-next/blob/52baa2167627d0cc88deb6858752492a1d9be7c5/cypress/integration/Integration.spec.js
- https://github.com/olyaboldina/cypress_yarn/blob/e1085dda7883fd13c1efb35ad8281d7e678f403c/cypress/integration/my_cypress.spec.js
- https://github.com/virtual-clothing/vr-store/blob/33b57fbb5ccd13f5f0ca21dc4307ff8521d9ae9c/cypress/integration/account_test_cypress.js
- https://github.com/rewddd/cypress-safari/blob/3f6af93d7fc8401408135c40e62f381f0ce94e88/cypress/integration/cypress-safari.spec.js
- https://github.com/0xCourtney/Food-Connect/blob/39bee460412d2f775e16b24fd54bf3ce609bdf77/cypress/integration/reagan_test_cypress.js
- https://github.com/OctoPrint/OctoPrint/blob/master/tests/cypress/integration/settings.js
- (No Semicolon) https://github.com/sridhar562345/cypress/blob/b09f770663d8e552721e3ddb3450737107e53b19/packages/driver/cypress/integration/cypress/cypress_spec.js

Reply:
Interestingly it looks like it's roughly 50/50 if the `it()` block should have a semicolon or not. I think I've added it for consistency with everything else having a semicolon but 🤷‍♂️